### PR TITLE
convert topology operator backed panel to use dynamic extensions

### PR DESCRIPTION
**Pull request by sahil143**
fixes: https://issues.redhat.com/browse/ODC-5846, task: https://issues.redhat.com/browse/ODC-6114

This PR 

- converts topology operator backed panel to use dynamic extensions 
- Migrate CSV default action to use new Action Provider extension
- Defined a new context `csv-actions` for operator backed
- migrated ceph-storage-plugin actions to use new action extensions

Screenshot:
image

### CODE REVIEW

**Review by vikram-raj — COMMENTED** [review]
verified it and it works as expected.

File: frontend/packages/ceph-storage-plugin/console-extensions.json [hunk 1]
```diff
@@ -198,5 +198,12 @@
         "$codeRef": "odfSystemDashboard.default"
       }
     }
+  },
+  {
+    "type": "console.action/provider",
+    "properties": {
+      "contextId": "csv-actions",
```
Comment (rohitkrai03) [inline frontend/packages/ceph-storage-plugin/console-extensions.json:205]: Any reason to use a new context id for CSVs? Wouldn't CSV resource actions work here?

**Review by rohitkrai03 — CHANGES_REQUESTED** [review]

File: frontend/packages/ceph-storage-plugin/console-extensions.json (hunk 1 above)
Comment (christianvogt) [reply frontend/packages/ceph-storage-plugin/console-extensions.json:205]: Why not use `console.action/resource-provider` extension?

Comment (sahil143) [reply frontend/packages/ceph-storage-plugin/console-extensions.json:205]: we have common actions `Delete, Edit yaml` that shows up on each operator backed resources that's why used a different context menu https://github.com/openshift/console/pull/9998/files#diff-9993fe676fb030dc047b550a63b46524797154b76f9ff27cf33d8abc24233081R11-R12

**Comment by invincibleJai** [comment]
/approve

verified the changes. I see missing i18n on list view `react_devtools_backend.js:4049 Missing i18n key "Operator Backed Service" in namespace "public" and language "en." `

This happens even on main, will be nice if we get it handled.

**Review by rottencandy — APPROVED** [review]
/lgtm

**Comment by sahil143** [comment]
Propagate from epic:

/label px-approved
/label docs-approved

**Pull request merged**